src/strings/en-us.json

```diff
@@ -97,7 +97,7 @@
     "ButtonMore": "More",
     "ButtonNextTrack": "Next track",
     "ButtonOk": "Ok",
-    "ButtonOpen": "Open",
+    "ButtonEditUser": "Edit user",
```

Comment:
Not a big deal since there are already a bunch that aren't, but preferably this would be in alphabetical order. :upside_down_face: